exercises/minesweeper/src/lib.rs

```diff
@@ -1 +1,3 @@
-
+pub fn annotate(minefield: &[&str]) -> Vec<String> {
+    unimplemented!("\nAnnotate the given minefield with the number of surrounding mines in each square (blank if there are no surrounding mines):\n{:#?}\n", minefield);
```

Comment:
can you confirm: does this look better with the leading newline?

Reply:
`cargo test` with newline:
```text
---- mine_surrounded_by_spaces stdout ----
thread 'mine_surrounded_by_spaces' panicked at 'not yet implemented: 
Annotate the given minefield with the number of surrounding mines in each square (blank if there are no surrounding mines):
[
    "   ",
    " * ",
    "   "
]
', src/lib.rs:2:5
```
`cargo test` without newline:
```text
---- mine_surrounded_by_spaces stdout ----
thread 'mine_surrounded_by_spaces' panicked at 'not yet implemented: Annotate the given minefield with the number of surrounding mines in each square (blank if there are no surrounding mines):
[
    "   ",
    " * ",
    "   "
]
', src/lib.rs:2:5
```

I would say that the newline is needed here, so that the helper text is positioned near the minefield 'image'

Reply:
👍 in that case, I'll ask whether `rectangles` would look better with a newline. if so, feel free to add it there as well, sorry I did not wait to give you a chance before merging `rectangles`